docs-source/content/userguide/managing-domains/model-in-image/debugging.md

```diff
@@ -99,3 +108,9 @@ Look for `SEVERE` and `ERROR` level messages in your operator logs. For example:
     | egrep -e "level...(SEVERE|WARNING)" \
     | grep "domainUID...sample-domain1"
   ```
+
+#### Check the FAQ
+
+Common issues with corresponding FAQ entries include:
```

Comment:
Common issues with corresponding FAQ entries include -> Common issues that have corresponding FAQ entries, include: (otherwise, the sentence could mean that there are issues with the FAQ.)